objects/body/include/object_body.h

```diff
@@ -10,10 +10,44 @@
 #ifndef OBJECT_BODY_H_
     #define OBJECT_BODY_H_
 
+    #include <stddef.h>
+
+    ///
+    /// \brief Max size of a body object
+    ///
+    #define BODY_SIZE 512
+
+    ///
+    /// \brief Error inside a body object
+    ///
+    #define BODY_ERROR NULL
+
+    ///
+    /// \brief Body error message for bad size
+    ///
+    #define BODY_INVALID_SIZE "Invalid content size."
+
 ///
 /// \brief Body object equivalent of a string
 ///
 ///
-typedef char body_t[512];
+typedef char *body_t;
+
+///
+/// \brief Init a Body object
+/// If the body_content is too long compared to BODY_SIZE, return BODY_ERROR
```

Comment:
It contains return value, move it in \return

Reply:
Fix in the next PR